### PR TITLE
Sync HRInfo docs

### PR DESCRIPTION
**Pull request by lazysoundsystem**
Replacing https://github.com/UN-OCHA/hrinfo-site/pull/971, this is based on the similar scripts for AR and KM.

There are some extra debugging messages, and there's a limit on the quantity of files, so as not to overload the staging site, but it adds a load of hrinfo documents and should be able to handle the rest.

It should maybe use a couple more vocabularies in common with AR, and there's some discussion to be had about which should be shared with other sites.

Some of the naming of fields in HRinfo is confusing, this sticks close to the names in the API, but the naming should probably be improved now to make it more clear in the future.

### CODE REVIEW

File: html/modules/custom/docstore/syncs/create_hrinfo_documents.php [hunk 1]
```diff
@@ -0,0 +1,426 @@
...
+
+function createVocabularies() {
+  $vocabularies = [
+    'Bundles' => 'hrinfo_bundles',
+    'Coordination hubs' => 'hrinfo_coordination_hubs',
+    'Document type' => 'hrinfo_document_type',
+    'Operations' => 'hrinfo_operations',
+    'Organizations' => 'hrinfo_organizations',
+    'Sectors' => 'hrinfo_sectors',
+    'Themes' => 'hrinfo_themes',
```
Comment (attiks) [inline html/modules/custom/docstore/syncs/create_hrinfo_documents.php:42]: Themes are a shared vocabulary

File: html/modules/custom/docstore/syncs/create_hrinfo_documents.php [hunk 2]
```diff
@@ -0,0 +1,426 @@
...
+  }
+}
+
+function createNodeType() {
+  // Document node type already created, nothing to do here.
+}
+
+function createVocabularies() {
+  $vocabularies = [
+    'Bundles' => 'hrinfo_bundles',
```
Comment (attiks) [inline html/modules/custom/docstore/syncs/create_hrinfo_documents.php:36]: These might be the same as local coordination groups

**Review by attiks — APPROVED** [review]
Looking good

**Comment by lazysoundsystem** [comment]
New changes to the hr-info script add some robustness and fix the references for local and global coordination groups.

The disaster references weren't matching as the disasters didn't have glide numbers - they need re-syncing before the hr-info documents are updated.

And the local_coordination_groups script also needs to run before. The 'activation document' field isn't included as the documents don't exist yet to be referenced. (These scripts are still more 'import' scripts than 'sync' script - they don't update existing terms. Do you think they should?)

Also, the themes vocabulary is for RW themes, not HRinfo ones - as themes are a little deprecated on HRinfo (moved to the hidden-by-default fields at the bottom of the form) I think it might be better to try to map them to the RW themes than to import all the HRinfo ones in a separate vocabulary. This needs more discussion.

**Pull request merged**